backend/chromeApiServer/src/main/java/com/canvas/service/SubmissionDirectoryService.java

```diff
@@ -105,6 +106,23 @@ private SubmissionFile[] writeSubmissionFiles(Map<String, byte[]> submissionFile
         return submissionFiles.toArray(new SubmissionFile[0]);
     }
 
+    public ResponseEntity<Deletion> deleteSubmissionDirectory(ExtensionUser user) {
+        String submissionDirectory = generateUniqueDirectoryName(user.getCourseId(), user.getAssignmentId(), user.getStudentId());
+        String description = user.getCourseId() + " " +  user.getAssignmentId() + " " + user.getStudentId() + " " + submissionDirectory;
+
+        // Now delete
+        fileService.deleteDirectory(submissionDirectory);
```

Comment:
there's a helper method in this class that already calls fileService to delete the directory